pkg/config/config.go

```diff
@@ -59,6 +43,10 @@ func NewFromFlags() (*Config, error) {
 
 	flag.Parse()
 
+	if _, exists := printer.Printers[config.Output]; !exists {
```

Comment:
Maybe we can use some function (align with `ParsePrinter`?)  instead of accessing the `.Printers` map directly. Making me think the `.Printers` could even be private, otherwise this is very tightly coupled.

Reply:
perfect! YES love this!